src/Controls/src/Xaml/Embedding/EmbeddingExtensions.cs

```diff
@@ -0,0 +1,41 @@
+using System;
+using System.Diagnostics.CodeAnalysis;
+using Microsoft.Maui.Controls.Hosting;
+using Microsoft.Maui.Hosting;
+
+namespace Microsoft.Maui.Controls.Embedding;
+
+/// <summary>
+/// A set of extension methods that allow for embedding a MAUI view within a native application.
+/// </summary>
+internal static class EmbeddingExtensions
```

Comment:
This also is internal for net8.0 and should be public for net9.0. This is also visible to the Community Toolkit.